deploy/managed-upgrade-operator-config/10-managed-upgrade-operator-configmap.yaml

```diff
@@ -8,7 +8,7 @@ data:
     upgradeType: OSD
     configManager:
       source: OCM
-      ocmBaseUrl: ${OCM_BASE_URL}
```

Comment:
What's the reasoning for hard-coding this instead of reusing the OCM_BASE_URL setting?

Reply:
We're not hard coding it @rendhalver, we're directing MUO to use OCM agent to communicate with OCM. OCM agent still uses `OCM_BASE_URL`:
https://github.com/openshift/managed-upgrade-operator/pull/394
```
    resources:
    - apiVersion: ocmagent.managed.openshift.io/v1alpha1
      kind: OcmAgent
      metadata:
        name: ocm-agent
        namespace: openshift-ocm-agent-operator
      spec:
        agentConfig:
          ocmBaseUrl: ${OCM_BASE_URL}
          services:
          - service_logs
          - clusters_mgmt
```

Reply:
Ah, never mind @rendhalver - it actually only works with `svc.cluster.local`. It seems that `.svc` only seems to work with the image registry? :thinking: